standards/rmrk2.0.0/entities/base.md

```diff
@@ -0,0 +1,200 @@
+            },
+            "wing_1_slot": {
+                "type": "slot",
+                "equippable": ["id-of-genesis-legendaries", "id-of-genesis-rares", "id-of-genesis-epics", ...],
+                "unequip": "burn",
+                "z": 2
+            }
+        }
+    ]
+}
+```
+
+## Base ID
+
+This is an arbitrary value, used for namespacing part slots. See Parts below. It can be any string
+value.
+
+## Base Type
+
+The type is a pre-defined value that clearly specifies how an NFT with a certain base type is
+supposed to be rendered. Currently, the only supported value is SVG:
+
+```json
+"type": "svg"
+```
+
+### SVG Base
+
+The parts will have Z indexes for layered rendering, and the final image is a composited, layered
+SVG. The SVG type implies the following rules:
+
+- every part will have a render in the same viewport
+- every part will be vector graphics
+- every part's 0,0 coordinates are in the top left corner
+
+This makes it possible to overlay SVG assets on top of each other without worrying about location in
+the rendering space. Example:
+
+![SVG composition](../images/svg_composition.png)
+
+The image above shows three different SVG assets, absolutely positioned within the same viewport.
+Notice how the viewport does not scale to bound the SVG asset, but instead scales to match the full
+size of the composited SVG (indicated by the faint bird outline). When combining these SVGs one on
+top of the other, the composited effect is achieved. This is possible because of part Z-index
+definitions (see below) and SVG transparency.
+
+#### SVG Base Parts
+
+`parts` is an array of Part objects. A Part object has two types: `fixed` and `slot`.
+
+**Fixed** parts are references to static content, like an IPFS hash of an SVG file, and a `z` index
+value, like so:
+
+```json
+  "wing_1_front": {
+      "type": "fixed",
+      "z": 3,
+      "src": "ipfs://ipfs/hash2"
+  },
+```
+
+The renderer will take the content behind the `src` value, and place it into the viewport at the Z
+index defined. The renderer does not check that the viewport of all parts matches - this is up to
+the designer (refer to the image above).
+
+**Slot** parts have a type of `slot` and no static resource `src`. They are meant to visually accept
+other NFTs into them. They have an array of whitelisted equippable collections, and an optional
+unequip value.
+
+The `equippable` value is a list of collections equippable into this slot. This value can also be a
+wildcard `*` to allow any collection to be equippable into this slot without special approval from
+the base owner. The whitelisting can be useful to prevent others from "hijacking" your project with
+their own customizations, covering all your art and branding.
+
+The optional `unequip` value specifies what happens to an item once it's unequipped. This is useful
+for one-off bonuses, NFTs that expire, and so on. The values can be `inv` for inventory and `burn`.

```

Comment:
nice, but do you think we will need many different types here or we can simply say `burnOnUnequip: boolean` or perhaps `multiuse: boolean`

Reply:
I think we might need others like "inc" or "dec" for limited-number-of-uses things etc, so I'd rather leave it open